README.md

```diff
@@ -1,3 +1,17 @@
+---
+
+This library, ADAL for .NET, will no longer receive new feature improvements. Instead, use the new library
+[MSAL for .NET](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet).
+
+* If you are starting a new project, you can get started with the
+  [MSAL for .NET docs](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki)
+  for details about the scenarios, usage, and relevant concepts.
```

Comment:
>scenarios [](start = 24, length = 9)

can you link to the scenario landing page here? [https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-overview#getting-started](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-overview#getting-started)

Reply:
For this and earlier comments, I'm going to say probably not since we want to be consistent across all ADAL repos. We can add this additional info in the main body of the readme if it isn't there already.

Reply:
@aiwangmicrosoft I see. Didn't know we had a blanket statement. thanks for clarifying.

Reply:
@aiwangmicrosoft @jennyf19 ; I think that we should have a page in docs.ms exposing the scenarios supported by MSAL.NET (that is all but SPA), and linking to the scenario landing page

That should be the case for all libraries, BTW.

Meanwhile we could link to https://aka.ms/msal-net-scenarios, which can be changed later